app/controllers/v0/medical_copays_controller.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module V0
+  class MedicalCopaysController < ApplicationController
+    before_action { authorize :medical_copays, :access? }
+
+    def index
```

Comment:
Do you have swagger updates for this new endpoint?

Reply:
Those are coming up next 👍  (Separate PR)